src/core/api/internal.go

```diff
@@ -172,7 +172,8 @@ func (ia *InternalAPI) SyncQuota() {
 			cfgMgr.Save()
 		}()
 		log.Info("start to sync quota(API), the system will be set to ReadOnly and back it normal once it done.")
-		err := quota.Sync(ia.ProjectMgr, false)
+		// As the sync function ignores all of duplicate error, it's safe to enable persist DB.
```

Comment:
could you please elaborate this a little bit?

Reply:
it just an enhancement for sync quota API, which is designed for sys-admin used only for some urgent case to correct the quota usage data.

Originally, it will not persist the artifact related data as a duplicate failure. Here just enable it as the duplicate errors are all ignored in the previous fix.